### PR TITLE
chore: fix minor issues in build scripts

### PR DESCRIPTION
**Pull request by danielroe**
## Types of changes

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description
This fixes a couple of minor issues (see GitHub comments).

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [x] All new and existing tests are passing.

### CODE REVIEW

File: scripts/package.js [hunk 1]
```diff
@@ -239,11 +294,6 @@ export default class Package {
             this.formatError(event.error)
             return this.logger.error(event.error)
 
-          // Encountered an unrecoverable error
-          case 'FATAL':
-            this.formatError(event.error)
-            return this.logger.fatal(event.error)
-
```
Comment (danielroe) [inline scripts/package.js:246]: There is no longer a FATAL event.

https://github.com/rollup/rollup/issues/3336

File: scripts/package [hunk 2]
```diff
@@ -27,7 +29,7 @@ async function main () {
   // Step 1: Apply suffixes
   for (const pkg of workspacePackages) {
     if (pkg.options.suffix && pkg.options.suffix.length) {
-      await pkg.suffixAndVersion()
+      pkg.suffixAndVersion()
```
Comment (danielroe) [inline scripts/package:32]: This is not an async method.

File: scripts/package.js [hunk 3]
```diff
@@ -329,8 +390,13 @@ export default class Package {
...
   }
 
+  /**
+   * @param {string} command
+   * @param {string} args
+   * @param {boolean} silent
+   */
   exec (command, args, silent = false) {
-    const r = spawn.sync(command, args.split(' '), { cwd: this.options.rootDir }, { env: process.env })
+    const r = spawn.sync(command, args.split(' '), { cwd: this.options.rootDir, env: process.env })
```
Comment (danielroe) [inline scripts/package.js:399]: `cross-spawn` doesn't take a third parameter

**Comment by pi0** [comment]
Thanks @danielroe for improvements ☺️ 

But i think having tsdocs for internal build scripts is unnecessary complexity until we migrate repo to typescript for nuxt3

**Review by TheAlexLichter — APPROVED** [review]

**Pull request merged**